sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -483,11 +482,8 @@ async def _read(self, n, buffer=None, **kwargs): # pylint: disable=unused-argume
 
     def close(self):
         """Do any preliminary work in shutting down the connection."""
-        # TODO: async close doesn't:
-        # 1) shutdown socket and close. --> self.sock.shutdown(socket.SHUT_RDWR) and self.sock.close()
```

Comment:
checking that we don't need to add `self.ws.shutdown(socket.SHUT_RDWR)` before the ws.close call?